pkg/cmd/operator/sidecar.go

```diff
@@ -38,6 +38,8 @@ type SidecarOptions struct {
 	ExternalSeeds                     []string
 	NodesBroadcastAddressTypeString   string
 	ClientsBroadcastAddressTypeString string
+	AdditionalScyllaDBArguments       []string
+	DeveloperMode                     bool
```

Comment:
I'd like to se this in our generated yaml config

Reply:
Unfortunately developer-mode is not only parameter to ScyllaDB, but also it's image entrypoint. When it's specified in the config and not as a parameter, iotune and additional hardware checks are executed and prevent Scylla process from starting.

Reply:
ok, we can get back to it when we start scylla directly